src/sage/arith/srange.pyx

```diff
@@ -84,7 +84,7 @@ def xsrange(start, end=None, step=1, universe=None, *, coerce=True, bint include
     EXAMPLES::
 
         sage: xsrange(10)
-        <generator object at 0x...>
+        <_cython_3_0_0.generator object at 0x...>
```

Comment:
```suggestion
        <_cython_3_....generator object at 0x...>
```
or maybe just
```suggestion
        <...generator object at 0x...>
```
since the original one is already failing with cython 3.0.2 (which I'm testing with right now).